src/components/taskDetails/ProgressUpdateCard/LatestProgressUpdateCardPresentation.tsx

```diff
@@ -65,7 +70,11 @@ export default function LatestProgressUpdateCardPresentation({
                 >
                     <Tooltip
                         content={tooltipText}
-                        tooltipPosition={{ top: '-2.6rem', right: '-4rem' }}
+                        tooltipPosition={
+                            isDevMode
+                                ? { top: '-3.2rem', right: '-2rem' }
+                                : { top: '-2.6rem', right: '-4rem' }
+                        }
```

Comment:
here also

Reply:
I have tried to reduce to use negative value as less I can with the latest commit